package.json

```diff
@@ -9,20 +9,13 @@
     "start": "ts-node src/index.ts"
   },
   "dependencies": {
-    "@ethersproject/abi": "^5.0.1",
     "@ethersproject/address": "^5.0.1",
-    "@ethersproject/constants": "^5.0.1",
-    "@ethersproject/contracts": "^5.0.1",
-    "@ethersproject/hash": "^5.4.0",
-    "@ethersproject/providers": "^5.3.1",
-    "@ethersproject/units": "^5.0.1",
     "@ethersproject/wallet": "^5.0.1",
     "@fleekhq/fleek-storage-js": "^1.0.15",
     "@pinata/sdk": "^1.1.14",
     "@pusher/push-notifications-server": "^1.2.4",
     "@snapshot-labs/snapshot.js": "^0.2.19",
```

Comment:
We'll have to update `snapshot.js` for the changes to work.

Reply:
Which version should I change to? 0.2.20?